erpnext/accounts/report/accounts_payable/accounts_payable.py

```diff
@@ -40,7 +40,9 @@ def execute(filters=None):
 
 			if abs(flt(outstanding_amount)) > 0.01:
 				paid_amount = invoiced_amount - outstanding_amount
-				row = [gle.posting_date, gle.account, gle.voucher_type, gle.voucher_no, 
+                		icon="""<a href="%s"><i class="icon icon-share" style="cursor: pointer;"></i></a>""" \
```

Comment:
The type of link with icon is not standard. 'Voucher no' column must have a link directly to the data, based on 'Voucher type' for this specific case. Please correct it.